Shared/Views/Buttons/RoundedMaterialButtonStyle.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  RoundedMaterialButtonStyle.swift
+//  QRiosity
+//
+//  Created by mrfour on 2021/12/26.
+//  Copyright © 2021 mrfour. All rights reserved.
+//
+
+import SwiftUI
+
+/// A button style that applies a large corner radius and material background based on the button's context.
+///
+/// - Note: This is currently the main style for buttons whose `label` is an icon. May have a more mature design system
+/// and have a better name in the next phase.
+struct RoundedMaterialButtonStyle: ButtonStyle {
+    enum Design {
+        static let shape = RoundedRectangle(cornerRadius: 24, style: .continuous)
+    }
+
+    func makeBody(configuration: Configuration) -> some View {
+        configuration.label
+            .frame(width: 24, height: 24)
+            // TODO: will have a better color for destructive later
```

Comment:
This comment is for trying the comment feature from **Xcode 13**.
```suggestion
            // TODO: will have a better color for destructive later.
```